lrauv_ignition_plugins/proto/lrauv_state.proto

```diff
@@ -33,66 +33,114 @@ import "ignition/msgs/header.proto";
-  float buoyancyPosition_         = 11;  // Volume of the VBS. Unit: cubic
-                                         // meters. Volumes higher than the
-                                         // neutral volume push the vehicle
-                                         // upwards.
+
+  /// \brief Position of the battery's joint. Unit: meters. Positive values have
+  /// the battery forward, tilting the nose downward.
+  float massPosition_             = 10;
+
+  /// \brief Volume of the VBS. Unit: cubic meters. Volumes higher than the
+  /// neutral volume push the vehicle upwards.
+  float buoyancyPosition_         = 11;
+
+  /// \brief Vertical position of the vehicle with respect to sea level. Higher
+  /// values are deeper, negative values are above water. Unit: meters.
   float depth_                    = 12;
 
-  ignition.msgs.Vector3d rph_     = 13;  // roll_, pitch_, heading_ in SimResultStruct (rad)
+  /// \brief Duplicate of posRPH_
+  ignition.msgs.Vector3d rph_     = 13;
+
+  /// \brief TODO(chapulina)
+  float speed_                    = 14;
 
-  float speed_                    = 14;  // Magnitude of linear velocity in the
-                                         // world frame. Unit: m / s
+  /// \brief Latitude in degrees
   double latitudeDeg_             = 15;
+
+  /// \brief Longitude in degrees
   double longitudeDeg_            = 16;
-  float netBuoy_                  = 17;  // Net buoyancy forces applied to the
-                                         // vehicle. Unit: Newtons. Currently
-                                         // not populated.
-
-  ignition.msgs.Vector3d force_   = 18;  // forceX_, forceY_, forceZ_ in SimResultStruct
-  ignition.msgs.Vector3d pos_     = 19;  // posX_, posY_, posZ_ in SimResultStruct
-  ignition.msgs.Vector3d posRPH_  = 20;  // posRoll_, posPitch_, posHeading_ in SimResultStruct
-  ignition.msgs.Vector3d posDot_  = 21;  // posXDot_, posYDot_, posZDot_ in SimResultStruct
-                                         // Velocity wrt ground
-  ignition.msgs.Vector3d rateUVW_ = 22;  // rateU_, rateV_, rateW_ in SimResultStruct
-                                         // Water velocity
-  ignition.msgs.Vector3d ratePQR_ = 23;  // rateP_, rateQ_, rateR_ in SimResultStruct
-                                         // for roll, pitch, yaw rates, respectively
-
-  float northCurrent_             = 24;  // +Y velocity in m / s
-  float eastCurrent_              = 25;  // +X velocity in m / s
-  float vertCurrent_              = 26;  // +Z velocity in m / s
+
+  /// \brief Net buoyancy forces applied to the vehicle. Unit: Newtons. Currently not populated.
+  float netBuoy_                  = 17;
+
+  /// \brief Not populated
+  ignition.msgs.Vector3d force_   = 18;
+
+  /// \brief \TODO(chapulina)
+  ignition.msgs.Vector3d pos_     = 19;
+
+  /// \brief \TODO(chapulina)
+  ignition.msgs.Vector3d posRPH_  = 20;
+
+  /// \brief \TODO(chapulina)
```

Comment:
```suggestion
  /// \brief Velocity wrt to ground \TODO(chapulina) check axes
```

Reply:
I removed these on purpose, I'm adding the clearer explanation on https://github.com/osrf/lrauv/pull/81/files#diff-638da727e840523f41a6bdea345acc02e4ab2b12ee06b88aa70ac15ae8b07654.